src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -35,7 +35,7 @@ class PexCli(TemplatedExternalTool):
     name = "pex"
     help = "The PEX (Python EXecutable) tool (https://github.com/pantsbuild/pex)."
 
-    default_version = "v2.1.137"
+    default_version = "v2.1.147"
     default_url_template = "https://github.com/pantsbuild/pex/releases/download/{version}/pex"
     version_constraints = ">=2.1.135,<3.0"
```

Comment:
AIUI, the minimum version here only needs to be bumped if pants itself relies on the new features/bug fixes in the PEX version. That doesn't seem to be the case here?

Reply:
Yeah. Iirc, his constraint functionality is meant to help people who host their own tools. We want to proactively error when they're using an old version of the tool.